tsconfig.json

```diff
@@ -19,7 +19,8 @@
     ],
     "paths": {
       "@/*": ["./src/*"]
-    }
+    },
+    "target": "ES2017"
```

Comment:
**Question:** Should we be targetting something more recent?  `.eslintrc.js` targets `es2021` and this file specifies some options as `esnext` above.

Reply:
Yeah, not sure why but Next was the one that did that when I changed it to next.config.ts, will check if esnext behaves fine